sdks/js/packages/core/react/components/onboarding/subscribe.tsx

```diff
@@ -1,111 +1,187 @@
 'use client';
 
-import { yupResolver } from '@hookform/resolvers/yup';
-import { Button, Flex, Text, Switch } from '@raystack/apsara/v1';
-import { Controller, useForm } from 'react-hook-form';
+import { ReactNode, useEffect, useState } from 'react';
 import * as yup from 'yup';
-import { Container } from '../Container';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { Button, Flex, Text, InputField, toast, ToastContainer, Image, EmptyState } from '@raystack/apsara/v1';
+import { useFrontier } from '~/react/contexts/FrontierContext';
+
+import PixxelLogoMonogram from '~/react/assets/logos/pixxel-logo-monogram.svg';
```

Comment:
lets not add pixxel logo here, instead we import this component and add the logo from there.